mantle/cmd/kola/testiso.go

```diff
@@ -80,10 +94,12 @@ RequiredBy=multi-user.target
 func init() {
 	cmdTestIso.Flags().BoolVarP(&instInsecure, "inst-insecure", "S", false, "Do not verify signature on metal image")
 	cmdTestIso.Flags().BoolVarP(&legacy, "legacy", "K", false, "Test legacy installer")
+	// TODO remove these --no-X args once RHCOS switches to the live ISO
 	cmdTestIso.Flags().BoolVarP(&nolive, "no-live", "L", false, "Skip testing live installer (PXE and ISO)")
 	cmdTestIso.Flags().BoolVarP(&nopxe, "no-pxe", "P", false, "Skip testing live installer PXE")
 	cmdTestIso.Flags().BoolVarP(&noiso, "no-iso", "", false, "Skip testing live installer ISO")
 	cmdTestIso.Flags().BoolVar(&console, "console", false, "Display qemu console to stdout")
+	cmdTestIso.Flags().StringSliceVar(&scenarios, "scenarios", []string{scenarioPXEInstall, scenarioISOInstall}, fmt.Sprintf("Test scenarios (also available: %s)", scenarioLegacyInstall))
```

Comment:
As a user, I think list default scenarios as well as available scenarios would be helpful, something like:
`fmt.Sprintf("Test scenarios (%s [default] %s [default] %s)", scenarioPXEInstall, scenarioISOInstall, scenarioLegacyInstall))`, though the description could be lengthy.

Reply:
I actually did that at first, but the library adds it by default:

```
walters@toolbox /v/s/w/s/g/c/coreos-assembler> ./mantle/bin/kola testiso --help | grep scena
      --scenarios strings   Test scenarios (also available: legacy-install) (default [pxe-install,iso-install])
walters@toolbox /v/s/w/s/g/c/coreos-assembler> 
```

Reply:
Aha, makes sense now :smiley:, resolving the comment